engine/runtime-integration-tests/src/test/java/org/enso/interpreter/runtime/ManagedResourceTest.java

```diff
@@ -7,29 +7,29 @@
 import static org.junit.Assert.assertNull;
 import static org.junit.Assert.assertTrue;
 
+import java.io.IOException;
 import java.lang.ref.Reference;
 import java.lang.ref.WeakReference;
 import org.enso.common.MethodNames;
-import org.enso.test.utils.ContextUtils;
+import org.enso.test.utils.ContextRule;
 import org.graalvm.polyglot.Context;
 import org.graalvm.polyglot.Source;
 import org.graalvm.polyglot.Value;
 import org.hamcrest.Matchers;
 import org.junit.AfterClass;
-import org.junit.BeforeClass;
+import org.junit.ClassRule;
 import org.junit.Test;
 
 public class ManagedResourceTest {
-  private static Context ctx;
-  private static EnsoContext ensoCtx;
+  @ClassRule
+  public static final ContextRule ctxRule =
+      ContextRule.newBuilder().initInContext(ManagedResourceTest::initCtx).build();
```

Comment:
I don't think you need `initInContext` at all. Use `@BeforeClass` for `@ClassRule` or `@Before` for `@Rule`.

Reply:
`initInContext` removed in 281602e6a5700f4b0fe3c35b1c7aa733ea284e5e